stubs/pyasn1/pyasn1/type/univ.pyi

```diff
@@ -28,15 +33,16 @@ class Integer(base.SimpleAsn1Type):
     def __rmul__(self, value): ...
     def __mod__(self, value): ...
     def __rmod__(self, value): ...
-    def __pow__(self, value, modulo: Incomplete | None = ...): ...
+    # Accepts everything builtins.pow does
```

Comment:
😱

Reply:
Luckily the Self return type means I can simplify all the overloads to `value: int | float | complex`. 😅